cmd/resolutioncli/main.go

```diff
@@ -150,22 +157,14 @@ func resolve(ctx context.Context, resolver *solver.DeppySolver, packageName stri
 	}
 
 	// Get the bundle image reference for the bundle
-	bundleImage, err := bundleEntity.BundlePath()
-	if err != nil {
-		return "", err
-	}
-
-	return bundleImage, nil
+	return bundleEntity.Image, nil
```

Comment:
Should probably rename `bundleEntity` -> `bundle`.

And below `getBundleEntityFromSolution` -> `bundleFromSolution` (in go we usually do not prefix with `get`).